api/api/test/test_validator.py

```diff
@@ -307,4 +307,4 @@ def test_check_component_configuration_pair(component, configuration, expected_r
         assert isinstance(response, expected_response.__class__)
         assert response.code == expected_response.code
     else:
-        assert response is expected_response
+        assert response is expected_response
```

Comment:
Add EOF line. You can configure VSCode so they are added automatically -> https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
Fixed in 8e322dd450925748ac5864038edb1c5146488d11.